doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu.rst

```diff
@@ -0,0 +1,126 @@
+  }
+
+  if [ -z "$1" ]
+  then
+      display_usage
+      exit 1
+  else
+      CONTAINER_NAME=$1
+      IMAGE_NAME=$2
+      NO_GPU=$3
+      if (docker ps --all | grep -q "$CONTAINER_NAME")
+      then
+          xhost +local:root &> /dev/null
+          echo "Found a docker container with the given name, starting $1"
+          printf "\n"
+          # If Docker is already running, no need to start it
+          if (docker ps | grep -q "$CONTAINER_NAME")
+          then
+              docker exec -it "$CONTAINER_NAME" /bin/bash && \
+              xhost -local:root 1>/dev/null 2>&1
+          else
+              docker start "$CONTAINER_NAME" 1>/dev/null 2>&1
+              docker exec -it "$CONTAINER_NAME" /bin/bash && \
+              xhost -local:root 1>/dev/null 2>&1
+          fi
+
+      else
+          if [ -z "$2" ]
+          then
+              printf "Can't find docker with the given name, need an image name to start the container from\n"
+              display_usage
+              exit 1
+          else
+              echo "Creating docker container $1 from image $2"
+              printf "\n"
+              if [ -z "$3" ]
+              then
+                  xhost +local:root &> /dev/null
+                  docker run -it --privileged \
+                      --net=host \
+                      --gpus all \
+                      --env=NVIDIA_VISIBLE_DEVICES=all \
+                      --env=NVIDIA_DRIVER_CAPABILITIES=all \
+                      --env=DISPLAY \
+                      --env=QT_X11_NO_MITSHM=1 \
+                      -v /tmp/.X11-unix:/tmp/.X11-unix \
+                      --name "$CONTAINER_NAME" \
+                      "$IMAGE_NAME" \
+                      /bin/bash
+                  xhost -local:root 1>/dev/null 2>&1
+              else
+                  # Start without GPU
+                  echo "Opening up the docker container without GPU support"
+                  printf "\n"
+                  xhost +local:root &> /dev/null
+                  docker run -it --privileged \
+                      --net=host \
+                      --env=DISPLAY \
+                      --env=QT_X11_NO_MITSHM=1 \
+                      -v "/tmp/.X11-unix:/tmp/.X11-unix" \
+                      --name "$CONTAINER_NAME" \
+                      "$IMAGE_NAME" \
+                      /bin/bash
+                  xhost -local:root 1>/dev/null 2>&1
+              fi
+          fi
+      fi
+  fi
+
+ There are 3 parameters for the script.
+  - name_of_the_container : this is the name you wish to give the created container
+  - name_of_the_image : if you are creating a fresh docker container, provide the name of the docker image here
+  - using_gpu : if `true`, the docker will be run using nvidia gpu drivers. By default, this value is true.
+
+3. Running the script
+
+To run the script and use nvidia gpu drivers
+::
+    ~/Docker/scripts/start-docker.sh moveit2-galactic moveit/moveit2:galactic-source
+
```

Comment:
Should have some instruction here on what to do if the above fails. They'll need to remove the container they just made to run the next command

```suggestion
If the above command fails, you are likely not using nvidia drivers. You'll need to remove the container you just created
`docker rm moveit2-galactic`
```